tendrl/commons/flows/__init__.py

```diff
@@ -115,7 +115,10 @@ def _execute_atom(self, atom_fqdn):
             ns, atom_name = atom_fqdn.split(".atoms.")
             ns, obj_name = ns.split(".objects.")
 
-            runnable_atom = self._ns.get_atom(obj_name, atom_name)
+            if hasattr(self, "obj"):
+                runnable_atom = self.obj._ns.get_atom(obj_name, atom_name)
+            else:
+                runnable_atom = self._ns.get_atom(obj_name, atom_name)
```

Comment:
"object._ns" : https://github.com/Tendrl/commons/blob/develop/tendrl/commons/__init__.py#L88
"atom._ns" : https://github.com/Tendrl/commons/blob/develop/tendrl/commons/__init__.py#L113
"flow._ns": https://github.com/Tendrl/commons/blob/develop/tendrl/commons/__init__.py#L124

@shtripat @nnDarshan could you guys re-check if this is still required, We do have the _ns object available for objects, atoms and flows